### DISCUSSION TITLE
My school wont let me download extensions for chrome recovery utility extension

### DISCUSSION THREAD

**Discussion by winghf** [3kh0/ext-remover#176] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Is there another way to downgrade the chromebook?

**Comment by ghost** [comment, marked as answer]
don't do it, you'll fuck up your school record and get banned from electronics

**Reply by 3kh0** [reply]
@winghf if you really want to know how, figure it out. Don't blame us if you get in trouble
